mtags/src/main/scala/scala/meta/internal/mtags/SymbolIndexBucket.scala

```diff
@@ -155,8 +126,15 @@ class SymbolIndexBucket(
         .map(_.symbol)
     val topLevels =
       if (source.isAmmoniteScript) sourceTopLevels.toList
-      else
-        sourceTopLevels.filter(sym => !isTrivialToplevelSymbol(uri, sym)).toList
+      else if (isJava) {
+        sourceTopLevels.toList.headOption
```

Comment:
Do you have any idea if the indexing is becoming much slower?

Reply:
```
Benchmark                      Mode  Cnt  Score   Error  Units
MetalsBench.javaMtagsPackage     ss   10  0.168 ± 0.014   s/op
MetalsBench.toplevelJavaMtags    ss   10  2.843 ± 0.112   s/op
```
It does get slower but it doesn't seem to be too bad. In practice for e.g. Metals there is no visible slowdown.

Reply:
Ok cool, let's merge it then :rocket:

Reply:
So the biggest slowdown I think is because we index JDK multiple times on one machine, which doesn't seem necessary. Should we have a separate database for JDKs then?